### DISCUSSION TITLE
'Find and replace' specific text in an event title

### DISCUSSION THREAD

**Discussion by beabadooboop** [derekantrican/GAS-ICS-Sync#408] [General]
One of the calendars I sync tends to put (TBC) at the start of an event title and I'd rather not see it as I don't need to know if it's confirmed yet or not.

Is there a way I can incorporate a bit of custom code to search for specific text (TBC) in the event titles and replace it with a specified string ("")?

If anyone has a solution, it'd be much appreciated!

**Comment by jonas0b1011001** [comment]
Sure, change
https://github.com/derekantrican/GAS-ICS-Sync/blob/17cde1a768ad4c6d5ede0e7401e20606fff0be2a/Helpers.gs#L441
to
`newEvent.summary = icalEvent.summary.replace("TBC","");`

**Comment by beabadooboop** [comment]
That's perfect, thankyou. Does exactly what I wanted.
